Mathlib/Algebra/Polynomial/AlgebraMap.lean

```diff
@@ -364,6 +364,10 @@ theorem aeval_X_left : aeval (X : R[X]) = AlgHom.id R R[X] :=
 theorem aeval_X_left_apply (p : R[X]) : aeval X p = p :=
   AlgHom.congr_fun (@aeval_X_left R _) p
 
+lemma aeval_X_left_of_algebra_apply [CommSemiring S] [Algebra R S] (p : R[X]) :
```

Comment:
I can't parse this name. Perhaps something like `aeval_X_left_eq_map` or `aeval_X_left_apply_eq_map_algebraMap` or somewhere in between?